src/db/schema.json

```diff
@@ -0,0 +1,29 @@
+{
+	"User": {
+		"_id": "ObjectID()",
+		"isBlind": "Boolean...if we want to tailor the experience based on how much vision one has. Like, if someone is blind, a map interface won't be useful to them. Just an idea!",
+		"firstName": "String",
+		"lastName": "String",
+		"email": "String",
+		"reviews": "Array<reviewIDs>"
+	},
+	"Review": {
+		"_id": "ObjectID()",
+		"userID": "ObjectID()",
+		"placeID": "Not sure what type this would be...the geoID? This will come from whatever API we use.",
+		"brailleRating": "Number",
+		"fontSizeRating": "Number",
+		"staffHelpfulnessRating": "Number",
+		"openessOfSpaceRating": "Number",
+		"comment": "String",
```

Comment:
do we want to have this all in one object or in many different objects? I guess it depends on how reviews work right? I thought that each review would be independent of each other (i.e., a user could rate say, openness of space without necessarily rating braille, or leave a numeric rating without leaving a comment). Or, perhaps these fields will just be null if the user has not interacted with them yet and we can keep the one big object for convenience

Reply:
I was thinking that they can be optional fields and default to null if they're not filled out